kernel-cflags-finder/Makefile

```diff
@@ -1,6 +1,7 @@
 ifneq ($(KERNELRELEASE),)
 obj-m += dummy.o
 clean-files := dummy.c
+ccflags-y += -Wno-unknown-warning-option
```

Comment:
What's this for?

Reply:
"Also, suppress one Clang warning class" - if you don't have it and you're using a new enough kernel, the build of the dummy module on a gcc-built kernel fails because clang doesn't recognize some `-W` flags that Linux v4.20 unconditionally passes, and the build is done with `-Werror`. `-Wunknown-warning-option` means "warn on unknown `-W` flags".